calculator.rb

```diff
@@ -0,0 +1,34 @@
+def prompt_operator
+  puts "enter a math operation"
```

Comment:
If I hadn't graded a few of these by now, I would've not known what the options were as a user ;) Consider adding in a nice print statement about what the valid operations are